index.html

```diff
@@ -114,6 +114,18 @@
       </ul>
     </section>
 
+    <section id='intent'>
+      <h2>Intent</h2>
+      <p>
+        The goal of this specification is to define the behaviors of Fedora server implementations in order to
+        facilitate interoperability with client applications.  Although Fedora servers are not drop-in replacements
```

Comment:
Two things:

1. I think it would be better to say "Although Fedora servers are not necessarily drop-in replacements" because you certainly could have two implementations that were that compatible.

2. There's a fair bit of hedging in the second part: "core behaviors", "stands to" and "a large subset".  I think it would be stronger and clearer to say that a client developed against the spec should work with any Fedora server with little modification.